README.md

```diff
@@ -4,7 +4,14 @@ A transport and encoding agnostic RPC framework.
 
 ## Installation
 
-`go get -u go.uber.org/yarpc`
+Use *at least* `glide version 0.12` to install:
```

Comment:
This is a bit weird. This suggests that you **have** to use glide. Suggest something like "If you use glide, make sure you're on at least version 0.12.0"